config.md

```diff
@@ -245,7 +245,7 @@ Presently there are `Prestart`, `Poststart` and `Poststop`.
 * [`Poststop`](#poststop) is a list of hooks to be run after the container process exits
 
 Hooks allow one to run code before/after various lifecycle events of the container.
-Hooks MUST be called in the listed order.
+Hooks MUST be called in the listed order if supported by the platform.
```

Comment:
On Tue, May 31, 2016 at 09:33:38AM -0700, Mrunal Patel wrote:
> > -Hooks MUST be called in the listed order.
> > +Hooks MUST be called in the listed order if supported by the platform.
>
> This is fine only if the bundle were to support different platforms
> else one could just not add hooks to the config at all.

The new text here isn't clear, but based on [1], I think @RobDolinMS
means “Windows-based runtimes will error on configuration files with
meaningful content in ‘hooks’ (e.g. anything more than empty arrays
for the per-event properties)”.  So for Windows this PR is aiming for
“you *can't* have hooks” not “you have a choice to not have hooks”.
The latter is already true, although it is made more explicit by the
in-flight #427.

[1]: https://github.com/opencontainers/runtime-spec/pull/472#issue-157263212